stardog/admin.py

```diff
@@ -46,6 +48,8 @@ def __init__(
                             username='admin', password='admin')
         """
         self.client = client.Client(endpoint, None, username, password, auth=auth)
+        # ensure the server is alive and at the specified location
+        self.healthcheck()
```

Comment:
So I think by calling this you are trying to get it to throw HTTPException if the server is unreachable, however you could still hit the case where you hit 4XX or 5XX and healthcheck will just return false. You need to check the value of healtcheck and raise if false.

Reply:
similar to my other comment, `healthcheck()` calls `Client.get()` which has the wrapper which handles the exception.

Reply:
sounds good